cli/lib/appdata.ts

```diff
@@ -15,8 +16,16 @@ const siteSchema = z
 	} )
 	.passthrough();
 
+const newSiteSchema = z
+	.object( {
+		id: z.string(),
+		path: z.string(),
+	} )
+	.passthrough();
+
 const userDataSchema = z
 	.object( {
+		newSites: z.array( newSiteSchema ).optional(),
```

Comment:
```suggestion
const userDataSchema = z
	.object( {
		newSites: z.array( siteSchema ).optional(),
```
In my quick testing, it looked like we could use `siteSchema` here, too. Maybe the current code is a leftover from a previous iteration?